pkg/dagger.io/dagger/core/exec.cue

```diff
@@ -125,3 +128,48 @@ import "dagger.io/dagger"
 #TempDir: {
 	size: int64 | *0
 }
+
+// Send a signal to a command created by #Start. SIGKILL is not
+// supported here. Instead, #Stop should be used to forcibly stop
+// a command.
+#SendSignal: {
+	$dagger: task: _name: "SendSignal"
+
+	input: #Start
+
+	signal: or([ for name, signal in _signals {signal}])
+}
+
+_signals: {
+	SIGHUP:    1
```

Comment:
> Nice, that's much cleaner! I also had to include a line that embedded _#signals on the top level of the file so outside callers could still say things like core.#SIGHUP, but that was easy too (thanks cue!)

Currently he will do `core.SIGHUP`, is it still ok for you?

Reply:
Ah yeah, I don't think making it a definition does anything since its just a concrete number.